x/subscription/spec/01_concepts.md

```diff
@@ -0,0 +1,29 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Subscription
+
+```go
+type Plan struct {
+  id int,  // unique identifier
+  owner Address,
+  price Coin,
```

Comment:
is it a fixed price or a max price?
for a floating exchange rate, one may want to be able to charge a different price each time

Reply:
merchant want to anchor to a fixed fiat currency price?

Reply:
yep, that's a very common use case

Reply:
maybe introduce some oracle?

Reply:
merchants generally won't do the collection on their own if they receive fiat and it's via a payment provider -- the payment provider does the collection and sets the rate

Reply:
but if the payment provider is malicious, he can increase the price by change the rate without subscriber's agreement?

Reply:
generally, that's set in the subscription terms and conditions -- if that happens, the subscriber can cancel subscription and if it broke the terms and conditions, they can go to a small claims court etc.

Reply:
done, payment provider or merchant can maintain the conversion rate.

Reply:
changed to fixed denomination.